### ISSUE TITLE
ERROR:    Error loading ASGI app. Could not import module "forge.app".

### ISSUE THREAD

**Issue by dhuang329** [Significant-Gravitas/AutoGPT#7225]
### ⚠️ Search for existing issues first ⚠️

- [X] I have searched the existing issues, and there is no existing issue for my problem

### Which Operating System are you using?

MacOS

### Which version of AutoGPT are you using?

Latest Release

### Do you use OpenAI GPT-3 or GPT-4?

GPT-3.5

### Which area covers your issue best?

Installation and setup

### Describe your issue.

After the asgi loading error was fixed, I deleted the whole folder and cloned AutoGPT again. Then I repeated these steps: added path environment variable so it can recognize the poetry package, changed the path of agbenchmark in pyproject.toml so it could find agbenchmark, set up OpenAI API key in .env. The error still happened. My device is a M1 Pro with macos Ventura 13.1.

**Comment by Swiftyos** [comment]
Have you pulled the latest version:

`git pull`

The try the following to activate the poetry env, which will handle making sure the deps are in the path:

```
poetry install
poetry shell
```

**Comment by dhuang329** [comment]
> Have you pulled the latest version:
> 
> `git pull`
> 
> The try the following to activate the poetry env, which will handle making sure the deps are in the path:
> 
> ```
> poetry install
> poetry shell
> ```

Hi, I tried to pull the latest version, but running poetry install gave this: 
Poetry could not find a pyproject.toml file in /Users/apple/AutoGPT or its parents
And I found the pyproject.toml was located in AutoGPT/autogpt and AutoGPT/agents/'myagentname'. So I cd to autogpt and run the two shell command. However, the error loading ASGI app. Could not import module "forge.app" still came up.
